src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -128,6 +129,28 @@ public function getConfigTreeBuilder(): TreeBuilder
                                 ->end()
                             ->end()
                         ->end() // end static
+                        ->variableNode('mime_types')
```

Comment:
this node should not be as a global option definitely, but more like under `static` arrayNode